server/repository/src/db_diesel/requisition/requisition_row.rs

```diff
@@ -156,13 +161,27 @@ impl<'a> RequisitionRowRepository<'a> {
     }
 
     #[cfg(not(feature = "postgres"))]
-    pub fn upsert_one(&self, row: &RequisitionRow) -> Result<(), RepositoryError> {
+    fn upsert_one_etc(&self, row: &RequisitionRow) -> Result<(), RepositoryError> {
```

Comment:
I'm not sure what the 'etc' is for - the fn is still just an upsert on the requisition row.

Reply:
tbh, i don't know what `_etc` stands for here, I saw it when Clemens used it in StorageConnection and liked it, my AI friend said it's to do with extra parameters (so it's incorrect):

![Screenshot 2023-05-11 at 3 11 08 PM](https://github.com/openmsupply/open-msupply/assets/26194949/27aa593d-7873-4218-b3ae-6e155a5ba430)

https://github.com/openmsupply/open-msupply/blob/c48435ae02d23d86ed19781570876004e17ab3bb/server/repository/src/db_diesel/storage_connection.rs#L79-L91

I think another convention is to use `_`, hope this is better: 26c6d8ac4ad09d2940f0ae3078fe9ce509aa34eb

Reply:
I also just used it because I saw it somewhere else before. The _etc function takes the full set of parameters and the other function is the more common version with a smaller number of params (to make the more common use case simpler).